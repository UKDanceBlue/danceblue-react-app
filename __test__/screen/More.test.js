import React from 'react';
import renderer from 'react-test-renderer';
import MoreScreen from '../../src/screens/More';

test('renders correctly', () => {
  const tree = renderer.create(<MoreScreen navigation={{ navigate: () => {} }} />).toJSON();
  expect(tree).toMatchSnapshot();
});
