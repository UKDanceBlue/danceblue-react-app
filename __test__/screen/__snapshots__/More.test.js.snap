// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#DDDDDD",
        "borderRadius": 10,
        "flex": 1,
        "height": "30%",
        "margin": 10,
      }
    }
  >
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "height": "100%",
          "opacity": 50,
          "width": "100%",
        }
      }
    >
      <Image
        source={1}
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": "100%",
              "width": "100%",
            },
            Object {
              "borderRadius": 10,
            },
          ]
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(0,0,0,0.5)",
            "borderRadius": 10,
            "bottom": 0,
            "flex": 1,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 50,
              "fontWeight": "bold",
            }
          }
        >
          Profile
        </Text>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#DDDDDD",
        "borderRadius": 10,
        "flex": 1,
        "height": "30%",
        "margin": 10,
      }
    }
  >
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "height": "100%",
          "opacity": 50,
          "width": "100%",
        }
      }
    >
      <Image
        source={1}
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": "100%",
              "width": "100%",
            },
            Object {
              "borderRadius": 10,
            },
          ]
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(0,0,0,0.5)",
            "borderRadius": 10,
            "bottom": 0,
            "flex": 1,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 50,
              "fontWeight": "bold",
            }
          }
        >
          About Us
        </Text>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#DDDDDD",
        "borderRadius": 10,
        "flex": 1,
        "height": "30%",
        "margin": 10,
      }
    }
  >
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "height": "100%",
          "opacity": 50,
          "width": "100%",
        }
      }
    >
      <Image
        source={1}
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": "100%",
              "width": "100%",
            },
            Object {
              "borderRadius": 10,
            },
          ]
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(0,0,0,0.5)",
            "borderRadius": 10,
            "bottom": 0,
            "flex": 1,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 50,
              "fontWeight": "bold",
            }
          }
        >
          Donate
        </Text>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#DDDDDD",
        "borderRadius": 10,
        "flex": 1,
        "height": "30%",
        "margin": 10,
      }
    }
  >
    <View
      accessibilityIgnoresInvertColors={true}
      style={
        Object {
          "height": "100%",
          "opacity": 50,
          "width": "100%",
        }
      }
    >
      <Image
        source={1}
        style={
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "height": "100%",
              "width": "100%",
            },
            Object {
              "borderRadius": 10,
            },
          ]
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(0,0,0,0.5)",
            "borderRadius": 10,
            "bottom": 0,
            "flex": 1,
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 50,
              "fontWeight": "bold",
            }
          }
        >
          FAQs
        </Text>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
